MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 include *.txt
 include *.md
-recursive-include docs
-recursive-include pydruid
+recursive-include docs *.txt
+recursive-include pydruid *.py
```

Comment:
I think `recursive-include pydruid *` is suffice. BTW I checked an all the files are `*.py`.